scss/_variables.scss

```
@@ -859,6 +859,7 @@ $form-file-button-hover-bg:       shade-color($form-file-button-bg, 5%) !default
 
 // scss-docs-start form-floating-variables
 $form-floating-height:            add(3.5rem, $input-height-border) !default;
+$form-floating-line-height:       1.25rem !default;
```

I think we can ditch the unit here. Tried in Safari and I saw no difference.

```suggestion
$form-floating-line-height:       1.25 !default;
```

---

Indeed!